docs/data/joy/main-features/color-inversion/color-inversion.md

```diff
@@ -21,21 +21,13 @@ The color inversion is implemented to solve this issue, keeping the global varia
 
 ## Overview
 
-When color inversion is enabled on the parent component, the children with implicit color will invert their styles to match the parent's background by keeping the same hierarchy of importance based on their variants.
+When color inversion is enabled on the parent component, all of Joy UI children will invert their styles regardless of its color prop to match the parent's background by keeping the same hierarchy of importance based on their variants.
 
 {{"demo": "ColorInversionOverview.js"}}
 
-**Implicit** color refers to components that don't have the `color` prop specified.
-
-Color inversion has no effect on children that have an **explicit** `color` prop.
-
-```js
-// implicit color. The styles change when color inversion is enabled.
-<Chip variant="soft">…</Chip>
-
-// explicit color. Color inversion has no effect.
-<Chip variant="soft" color="primary">…</Chip>
-```
+:::info
```

Comment:
Not a deal breaker for this PR, but we have too many info callouts on this page. Every second paragraph feels like a callout. I would rather convert them to text, as otherwise they don't serve the purpose to "call out".